### ISSUE TITLE
removed from CRAN

### ISSUE THREAD

**Issue by lolow** [JGCRI/RCMIP5#152]
A quick search on CRAN gives:

```
Package ‘RCMIP5’ was removed from the CRAN repository.

Formerly available versions can be obtained from the archive.

Archived on 2020-06-15 as check problems were not corrected in time.

Please use the canonical form https://CRAN.R-project.org/package=RCMIP5 to link to this page.
```

As some of my script rely on this package, I would like to know if the package is still maintained? 
Thanks!

**Comment by bpbond** [comment]
Hi @lolow - thanks for the heads-up. We have not been actively maintaining this package, no. As a temporary workaround are you able to install directly from GitHub, e.g. using `devtools::install_github()`?

@ktoddbrown as maintainer are you aware of this?

**Comment by lolow** [comment]
thanks for the reply. I managed to install the package using `remotes::install_github()`.

**Comment by ktoddbrown** [comment]
Yes I am aware that this was likely, there was a tidyverse update that had some backwards compatibility issues I believe. as you said @bpbond it's not actively being maintained. Although I wasn't aware that it was being used @lolow! I will take a look at updating the package over the next week or so.
